scripts/subsystem_versions.py

```diff
@@ -0,0 +1,68 @@
+"""This script is used to get the versions of the submodules."""
+
+import re
+import sys
+import json
+import argparse
+import subprocess
+
+VERSION_REGEX = re.compile("v([0-9])")
+SUBSYSTEMS = [
+    "head",
+    "gantry-x",
+    "gantry-y",
+    "gripper",
+    "pipettes-single",
+    "pipettes-multi",
+    "pipettes-96",
+    "pippetes-384",
+]
+
+
+def main(args):
+    version_info = {}
+    subsystems = args.target or SUBSYSTEMS
+
+    try:
+        version = subprocess.check_output(["git", "describe", "--tags", "--always", "--match=v*"]).decode().strip()
+        # version should be int
+        version = int(re.search(VERSION_REGEX, version).group(1)) if re.search(VERSION_REGEX, version) else 0
+        shortsha = subprocess.check_output(["git", "rev-parse", "--short", "HEAD"]).decode().strip()
+        branch = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).decode().strip()
+    except Exception as e:
+        print(f"Could not get the version info for {subsystem}", file=sys.stderr)
```

Comment:
`subsystem` isn't initialized at this point

Reply:
good catch, leftover from before refactor.